pip/req/req_file.py

```diff
@@ -51,7 +51,8 @@
 # options to be passed to requirements
 SUPPORTED_OPTIONS_REQ = [
     cmdoptions.install_options,
-    cmdoptions.global_options
+    cmdoptions.global_options,
+    cmdoptions.hash,
 ]
```

Comment:
I suppose one could, but it doesn't buy you much: specifying any `--hash` option at all in the file accomplishes the same thing.